assets/js/modules/ads/index.js

```diff
@@ -78,3 +80,19 @@ export const registerModule = ( modules ) => {
 		} );
 	}
 };
+
+export const registerWidgets = ( widgets ) => {
+	if ( isFeatureEnabled( 'adsPax' ) ) {
+		widgets.registerWidget(
+			'partnerAds',
```

Comment:
Also the widget slug, to something like `pax`? WDYT?

Reply:
Works for me 👍🏻